src/app/shared/model/series.model.ts

```diff
@@ -36,9 +37,8 @@ export class SeriesModel extends BaseModel implements HasUpload {
   constructor(account: HalDoc, series?: HalDoc, loadRelated = true) {
     super();
     this.init(account, series, loadRelated);
-    if (account) {
```

Comment:
I dunno why or if we need to support this, but the series.component used to init the SeriesModel with a null account. I changed that one and don't see it anywhere else, but if account is null/undefined and not checked, this will throw an error.